geode-gfsh/src/integrationTest/java/org/apache/geode/management/internal/cli/commands/AlterDiskStoreCommandIntegrationTest.java

```diff
@@ -55,4 +59,33 @@ public void removeOptionMustBeUsedAlone() {
     gfsh.executeAndAssertThat(command, commandString).statusIsError()
         .containsOutput("Cannot use the --remove=true parameter with any other parameters");
   }
+
+  @Test
+  public void testDirValidation() throws IOException {
```

Comment:
Can you remove `throws IOException` as it's not thrown from within the method body?.